.githooks/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+files=$(git diff --cached --name-status)
+
+for file in $files; do
+	if [ "${file##*.}" = "pkl" ]; then
+		pkl eval "$file" -o "${file%.pkl}.yml"
```

Comment:
What happens if someone doesn't have a global install of `pkl` on their machine?

Reply:
It would fail and the user can't commit 🫠

Reply:
Haha sure, I know how commit hooks work - I mean, `-bash: pkl: command not found` is a very weird error message to receive when you run `git commit`, not the best contributor experience.